hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/execution/OMExecutionFlow.java

```diff
@@ -36,10 +40,12 @@ public class OMExecutionFlow {
 
   private final OzoneManager ozoneManager;
   private final OMPerformanceMetrics perfMetrics;
+  private final OmLockOpr omLockOpr;
```

Comment:
Let's don't change OMExecutionFlow and OMClientRequest in this PR.  It is good to make it clear that this PR is not changing the existing code.